netrc_test.go

```diff
@@ -0,0 +1,93 @@
+package netrc
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"strings"
+	"testing"
+)
+
+func TestParse(t *testing.T) {
+	tcs := []struct {
+		content string
+		machine string
+		err     error
+		want    Entry
+	}{
+		{`machine mail.google.com
+	login joe@gmail.com
+	account gmail
+	password somethingSecret`, "mail.google.com", nil, Entry{"joe@gmail.com", "somethingSecret", "gmail"}},
```

Comment:
Please do not use real world domains for testing. Refer to [RFC 2606, Section 2](https://tools.ietf.org/html/rfc2606#section-2) as well as [RFC 6761, Section 6.2](https://tools.ietf.org/html/rfc6761)